docs/setup/configurations/configurations-overview.md

```diff
@@ -5,8 +5,8 @@ The configuration allows you to quickly roll back a configuration change if nece
 
 There are two ways you can perform configurations while setting up Devtron dashboard:
 
-* [Installation Configurations](setup/install/installation-configuration.md)
-* [Override Configurations](setup/install/override-default-devtron-installation-configs.md)
+* [Installation Configurations](https://github.com/devtron-labs/devtron/blob/main/docs/setup/install/installation-configuration.md)
```

Comment:
The link will still be broken as its coming from Github (after merging to main). Can you please add the link from the document itself? It is: https://docs.devtron.ai/configurations-overview/installation-configuration